class ProbeError(Exception):
    pass

class UnauthorizedError(Exception):
    pass

